Docs/sphinx_documentation/source/AMReX_Profiling_Tools.rst

```diff
@@ -324,8 +353,9 @@ the Tiny Profiler output and to allow instrumentation strategies without needing
 
 .. admonition:: A Combined Tiny-Full Region Profiling Strategy
 
-  We recommend using scoped regions in a few select places that will be timed when either the
-  Tiny or Full Profiler is activated. Because named regions will be isolated by the Full Profiler
+  We recommend using scoped regions in a few select places that will be isolated for analysis
+  when either the Tiny or Full Profiler is activated.
+  Because only named regions will be isolated by the Full Profiler
```

Comment:
I wasn't able to get this to work for myself, so I want to verify that "only named regions will be isolated by the Full Profiler... " is correct?